CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+Fix: macOS can run `buildkite_agent_count` job-runners now.
```

Comment:
```suggestion
* Fix: all platforms will run `buildkite_agent_count` job-runners now, defaulting to `1`.
* Fix: macOS `buildkite_agent_executable` is correct for the homebrew install now.
```

Reply:
Heh, I just pushed before reading this! Let's use this one!